src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -95,15 +106,21 @@ public void findDescriptor_emptyPrefixInput_emptyPredicatesList() throws ParseEx
         findDescriptor = new FindDescriptor(argMultimap);
         assertTrue(findDescriptor.getPredicates().isEmpty());
 
+        String emptyExperiencePrefixInput = " y/";
+        argMultimap = ArgumentTokenizer.tokenizeWithoutPreamble(emptyExperiencePrefixInput,
+                PREFIX_EXPERIENCE);
+        findDescriptor = new FindDescriptor(argMultimap);
+        assertTrue(findDescriptor.getPredicates().isEmpty());
+
         String emptyTagPrefixInput = " t/";
         argMultimap = ArgumentTokenizer.tokenizeWithoutPreamble(emptyTagPrefixInput, PREFIX_TAG);
         findDescriptor = new FindDescriptor(argMultimap);
         assertTrue(findDescriptor.getPredicates().isEmpty());
 
-        String multipleEmptyPrefixInput = " n/ e/ p/ r/ et/ s/ l/ t/";
+        String multipleEmptyPrefixInput = " n/ e/ p/ r/ et/ s/ l/ t/ y/";
         argMultimap = ArgumentTokenizer.tokenizeWithoutPreamble(multipleEmptyPrefixInput,
                 PREFIX_NAME, PREFIX_EMAIL, PREFIX_PHONE, PREFIX_ROLE, PREFIX_EMPLOYMENT_TYPE,
-                PREFIX_EXPECTED_SALARY, PREFIX_LEVEL_OF_EDUCATION, PREFIX_TAG);
+                PREFIX_EXPECTED_SALARY, PREFIX_LEVEL_OF_EDUCATION, PREFIX_TAG, PREFIX_EXPERIENCE);
```

Comment:
I think we should keep the order consistent, so `PREFIX_EXPERIENCE` should come right before `PREFIX_TAG`